server/onboardingimports/imports.go

```
@@ -0,0 +1,635 @@
+	out := make(map[string]bool, len(values))
+	for key, value := range values {
+		out[key] = value
+	}
+	return out
+}
+
+func normalizeName(raw string) string {
+	return strings.ToLower(strings.Join(strings.Fields(raw), " "))
+}
+
+func unixMs(t time.Time) *int64 {
```

golangci-lint (2.12.2)

[error] func unixMs is unused

(unused)